### ISSUE TITLE
problem with TutorialTest

### ISSUE THREAD

**Issue by rob-c** [ganga-devs/ganga#503]
I'm having a problem with testing ```Ganga/test/GPI/TutorialTests.py::TutorialTests::test_c_JobManipulation``` on a fast tests system as the test attempts to resubmit a job which is still in the submitted state.

Adding a line to at least kill the job or pause it until it's completed is enough to fix this but I'm aware this is part of the tutorial at some level.

Where is the best place to edit this to fix the problem?
(This really looks like it only hits my system as it's working everywhere else I've run)

**Comment by milliams** [comment]
Anything in `TutorialTests.py` between `START` and `STOP` sections is being displayed in the tutorial text. Anything outside those sections is feww to be used for background stuff like this. I would try changing those line to be like:
```python
# -- JOBMANIPULATION JOBSLICING STOP

jobs(0).kill()  # make sure the job can be resubmit
# -- JOBMANIPULATION RESUBMIT START
jobs(0).resubmit()
# -- JOBMANIPULATION RESUBMIT STOP
```

**Comment by rob-c** [comment]
How do these end up in the tutorials, is the code referenced from here or is this code autogenerated, I'm always slightly offput touching stuff which has a lot of shouting text.

**Comment by milliams** [comment]
If you look at [`ganga/develop/doc/UserGuide/JobManipulation.rst`](https://raw.githubusercontent.com/ganga-devs/ganga/develop/doc/UserGuide/JobManipulation.rst) you'll see bits like:
```rst
.. literalinclude:: ../../python/Ganga/test/GPI/TutorialTests.py
    :start-after: # -- JOBMANIPULATION RESUBMIT START
    :end-before: # -- JOBMANIPULATION RESUBMIT STOP
    :dedent: 8
```
which defines which file the snippet is sourced from, which markers to use for finding the snippet and how much to dedent the code.

This ends up as http://ganga.readthedocs.io/en/latest/UserGuide/JobManipulation.html#resubmitting-jobs

To make sure that you haven't broken anything, go to the `doc` directory of ganga and run `make html` to generate the docs.

**Comment by rob-c** [comment]
I cannot run the ```make html``` and the last time I did I ended up simply re-installing Ganga due to a lot of in-situe file modifications/replacements from a make command in this dir.
As long as it's safe for me to edit the test directly then I'm happy, as for what the documentation technically does I simply trust it works because every time I touch this I get burnt so I'm getting on with other tasks.

Good to know this is just as simple as fix/edit the test.
